--- original/src/parser/CVC.y	2013-12-12 11:13:18.000000000 -0500
+++ stp-r940/src/parser/CVC.y	2015-05-14 23:45:23.489546000 -0400
@@ -22,7 +22,6 @@
 #define YYMAXDEPTH 1048576000
 #define YYERROR_VERBOSE 1
 #define YY_EXIT_FAILURE -1
-#define YYPARSE_PARAM AssertsQuery
   
   extern int cvclex(void);
   extern char* yytext;
@@ -32,9 +31,12 @@
     FatalError("");
     return YY_EXIT_FAILURE;
   };
+  int yyerror(void* AssertsQuery, const char* s) { return yyerror(s); }
   
   %}
 
+%parse-param {void* AssertsQuery}
+
 %union {
 
   unsigned int uintval;                 /* for numerals in types. */
