--- original/src/parser/smtlib.y	2013-12-12 11:13:18.000000000 -0500
+++ stp-r940/src/parser/smtlib.y	2015-05-14 23:46:01.744665000 -0400
@@ -54,15 +54,17 @@
     FatalError("");
     return 1;
   }
+  int yyerror(void* AssertsQuery, const char* s) { return yyerror(s); }
 
   ASTNode query;
 #define YYLTYPE_IS_TRIVIAL 1
 #define YYMAXDEPTH 104857600
 #define YYERROR_VERBOSE 1
 #define YY_EXIT_FAILURE -1
-#define YYPARSE_PARAM AssertsQuery
   %}
 
+%parse-param {void* AssertsQuery}
+
 %union {  
   // FIXME: Why is this not an UNSIGNED int?
   int uintval;                  /* for numerals in types. */
